azurerm/resource_arm_virtual_machine.go

```diff
@@ -374,6 +374,11 @@ func resourceArmVirtualMachine() *schema.Resource {
 							Optional: true,
 							Default:  false,
 						},
+						"timezone": {
+							Type:         schema.TypeString,
+							Optional:     true,
```

Comment:
in the Azure API IIRC this defaults to `UTC` - we should default that here too?

Reply:
does this also need to be ForceNew, since IIRC this value is used as part of the initial post-sysprep?

Reply:
if not - can we add an example for updating a VM's TimeZone from UTC -> PST?

Reply:
Tried with Azure, it has to be `ForceNew`.